### PR TITLE
Db migration

### PR DESCRIPTION
**Pull request by liamrberney**
I created some basic db migrations; I had some trouble creating Enums for contract_type, contract_status, and time_period so I left them as integers for now.

### CODE REVIEW

File: db/migrate/20230219040609_create_contracts.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+class CreateContracts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :contracts do |t|
+      t.string :title
+      t.integer :number
```
Comment (cpepi001) [inline db/migrate/20230219040609_create_contracts.rb:5]: This needs to be string

File: app/models/movie.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class Movie < ApplicationRecord
```
Comment (cpepi001) [inline app/models/movie.rb:1]: We don't need that :)

Comment (liamrberney) [reply app/models/movie.rb]: My bad, fixed

**Review by cpepi001 — APPROVED** [review]

**Comment by matanbroner** [comment]
@liamrberney we shouldn't be making PRs too inflated. This PR was for DB migrations and was already approved. 
The UI changes should be a separate PR.

**Pull request closed without merging**